activejob/README.md

```diff
@@ -2,8 +2,8 @@
 
 Active Job is a framework for declaring jobs and making them run on a variety
 of queuing backends. These jobs can be everything from regularly scheduled
-clean-ups, to billing charges, to mailings. Anything that can be chopped up into
-small units of work and run in parallel, really.
+clean-ups, to billing charges, to mailings - anything that can be chopped up into
```

Comment:
Nit-pick: this is using a hyphen / minus sign.  It should be an em dash:

```suggestion
clean-ups, to billing charges, to mailings — anything that can be chopped up into
```

Reply:
@jonathanhefner review changes updated